src/test/java/com/google/sps/integration/ExampleIT.java

```diff
@@ -3,26 +3,55 @@
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
 import java.net.URL;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
Try to avoid * if possible. Do you know what you're trying to import specifically here?

Reply:
Done.  I think Intellij automatically turned it into *;  I suspect it's because I might be importing all of the packages under org.junit.*?